src/rust/ide/lib/view/src/graph_editor/builtin/visualization/native/documentation.rs

```diff
@@ -0,0 +1,184 @@
+//! Documentation view visualization generating and presenting Enso Documentation under
+//! the documented node.
+
+use crate::prelude::*;
+
+use crate::graph_editor::component::visualization;
+use crate::graph_editor::component::visualization::Instance;
```

Comment:
I was asking in previous PR to NOT import these things but use them in qualified form in the code. Right now you have code `impl From<View> for Instance { ... }` which is completely unredable. IT should be `impl From<View> for visualization::Instance { ... }` instead.

Reply:
oh i get it now 👍